### PR TITLE
Use rich to render info and cheat-sheet command

### PR DESCRIPTION
**Pull request by turbaszek**
### airflow info
Before:
Screenshot 2020-11-28 at 16 42 32

After:
Screenshot 2020-11-28 at 16 41 34

### airflow cheat-sheet
Before:
Screenshot 2020-11-28 at 19 07 52

After:
Screenshot 2020-11-28 at 19 46 18

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by mik-laj** [comment]
This documentation contains sample output from this command. Can you update it?
http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/modules_management.html

**Comment by turbaszek** [comment]
> http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/modules_management.html

Sure, is it only for me or the images are not rendering on this page?

**Comment by mik-laj** [comment]
I have images on this page.
Screenshot 2020-11-28 at 17 15 29

**Comment by turbaszek** [comment]
Screenshot 2020-11-28 at 17 21 43

**Comment by mik-laj** [comment]
There is a problem with one image because it references a file from airflow-site, but it shouldn't take up that much space as it's only used by anchor.
Screenshot 2020-11-28 at 17 30 09

File: airflow/cli/commands/info_command.py [hunk 1]
```diff
@@ -447,8 +418,7 @@ def _send_report_to_fileio(info):
     try:
         link = _upload_text_to_fileio(str(info))
         print("Report uploaded.")
-        print()
-        print("Link:\t", link)
+        print(link)
```
Comment (turbaszek) [inline airflow/cli/commands/info_command.py:409]: This simplifies output so users can do `airflow info --file-io | grep https | pbcopy`. Example:
```
Uploading report to file.io service.
Report uploaded.
https://file.io/xfDPsSSxkFm9
```

**Review by kaxil — APPROVED** [review]
❤️

**Review by potiuk — APPROVED** [review]
:heart:

File: airflow/cli/commands/cheat_sheet_command.py [hunk 2]
```diff
@@ -15,53 +15,44 @@
...
+    display_commands_index()
 
 
 def display_commands_index():
     """Display list of all commands."""
 
-    def display_recursive(prefix: List[str], commands: Iterable[Union[GroupCommand, ActionCommand]]):
+    def display_recursive(
+        prefix: List[str], commands: Iterable[Union[GroupCommand, ActionCommand]],
+        help_msg: Optional[str] = None
```
Comment (kaxil) [inline airflow/cli/commands/cheat_sheet_command.py]: This should fix CI failure

```suggestion
        prefix: List[str], 
        commands: Iterable[Union[GroupCommand, ActionCommand]],
        help_msg: Optional[str] = None
```

Comment (turbaszek) [reply airflow/cli/commands/cheat_sheet_command.py]: Yeah, thanks! I fixed it locally but did not add this to commit 🤦

File: docs/modules_management.rst [hunk 3]
```diff
@@ -232,29 +232,45 @@ Below is the sample output of the ``airflow info`` command:
...
+    system_path     | /opt/bats/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    python_path     | /usr/local/bin:/opt/airflow:/files/plugins:/usr/local/lib/python38.zip:/usr/local/lib/python3.8:/usr/
+                    | local/lib/python3.8/lib-dynload:/usr/local/lib/python3.8/site-packages:/files/dags:/root/airflow/conf
+                    | ig:/root/airflow/plugins
+    airflow_on_path | True
+
+    Config info
+    executor             | LocalExecutor
+    task_logging_handler | airflow.utils.log.file_task_handler.FileTaskHandler
+    sql_alchemy_conn     | postgresql+psycopg2://postgres:airflow@postgres/airflow
```
Comment (XD-DENG) [inline docs/modules_management.rst:266]: If everything is working correctly, the sample value here should be something like `postgresql+psycopg2://p...s:PASSWORD@postgres/airflow` instead, according to https://github.com/apache/airflow/blob/b858683abf5b17ef85d4b246e05871819a747f14/airflow/cli/commands/info_command.py#L85

May you please help double check?

OR, personally I think we can consider removing `sql_alchemy_conn` from the info command output

ADD-UP: if we show it here, either we are showing something useful BUT sensitive, or we are showing something not very useful (after sufficient necessary masking), IMHO

Comment (turbaszek) [reply docs/modules_management.rst:266]: In my opinion it works as expected. I can use example generated with `--anonymize` flag:
```
root@1aa14208df15:/opt/airflow# airflow info --anonymize
...
Config info
executor             | LocalExecutor
task_logging_handler | airflow.utils.log.file_task_handler.FileTaskHandler
sql_alchemy_conn     | postgresql+psycopg2://p...s:PASSWORD@postgres/airflow
```

If some one can run `airflow info` in your deployment - it's too late, so masking information like conn uri (which is also accessible via `airflow config` command) will not increase security.

Comment (XD-DENG) [reply docs/modules_management.rst:266]: My bad that I missed the `--anonymize` flag.

Regarding whether to keep `sql_alchemy_conn` in the output of `airflow info`, I still keep my view given it's (almost) the only sensitive item in the output. *But I will not insist in this opinion in this PR*.

OR, at least, let's make anonymisation default. Currently it's `store_true`, i.e. no anonymisation by default. Let me know your thoughts?

https://github.com/apache/airflow/blob/07a2ed5a2ab6edf3fc38c94186fe2204a47f44b7/airflow/cli/cli_parser.py#L649-L653

Comment (XD-DENG) [reply docs/modules_management.rst:266]: But can be discussed & addressed in separate PR anyway, if you prefer. I will approve this PR first.

**Review by XD-DENG — APPROVED** [review]

File: docs/modules_management.rst (hunk 3 above)
Comment (turbaszek) [reply docs/modules_management.rst:266]: Created a separate issue: https://github.com/apache/airflow/issues/12696

**Pull request merged**